src/mesh/Mesh.hpp

```diff
@@ -247,11 +263,17 @@ class Mesh {
   void tagAll();
 
   /// Returns a vector of connected ranks
-  std::vector<Rank> &getConnectedRanks()
+  const std::vector<Rank> &getConnectedRanks() const
```

Comment:
Please use a consistent `const` style (see other functions above)

Reply:
Is it? A few lines further up you write `std::vector<int> const &getVertexOffsets() const`. So I would expect here 
```suggestion
  std::vector<Rank> const &getConnectedRanks() const
```

Reply:
Ah correct, the ones below are wrong.

I see that clang-format 12 has an option for this. Shall we update?

Reply:
> I see that clang-format 12 has an option for this. Shall we update?

Well, that's on the list since longer, but doing this now with so many open PRs is probably not a good idea.